src/app/bridge/Send/SendTransaction/TokenList.tsx

```diff
@@ -235,7 +235,7 @@ const OptimizedComponent = ({ listedbyUser, listedbyScroll, selectToken, selecte
     )}
     {listedbyScroll?.length > 0 && (
       <>
-        <ListTitleStyled>Listed by t1</ListTitleStyled>
+        <ListTitleStyled>Listed by 𝚝𝟷</ListTitleStyled>
```

Comment:
Please, review the strings related to this. I think there are some more to change in two index files from the bridge. 

Aside from this, I discourage to keep using this particular `t`. It is going to be a headache and I'm not sure how much value adds.

Reply:
> Please, review the strings related to this. I think there are some more to change in two index files from the bridge.

Do you mean reuse this as a var? Can do, but that is a larger refactor than I have scoped for this PR.

> Aside from this, I discourage to keep using this particular t. It is going to be a headache and I'm not sure how much value adds.

Will defer to @cankisagun on this.

Reply:
Got it. So what is blocking your approval of this PR?